.github/workflows/nightly.yml

```diff
@@ -0,0 +1,58 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron: '0 1 * * *'   # everyday at 02:00 UTC+1
+  workflow_dispatch:
+
+jobs:
+  build-macos:
+    name: build-macos
+    runs-on: macos-12
+    strategy:
+      matrix:
+        target: [ia32-generic-qemu]
+        include:
+          - toolchain: i386-pc-phoenix
```

Comment:
I added toolchain to matrix. In the future it will be easier to add other compilers.